manifests/install.yaml

```diff
@@ -257,6 +257,30 @@ spec:
               key: webhook.harbor-secret
               name: argocd-image-updater-secret
               optional: true
+        - name: ENABLE_WEBHOOK_RATELIMIT
+          valueFrom:
+            configMapKeyRef:
+              key: webhook.enable-rate-limit
+              name: argocd-image-updater-config
+              optional: true
+        - name: WEBHOOK_RATELIMIT_NUM_ALLOWED_REQUESTS
+          valueFrom:
+            configMapKeyRef:
+              key: webhook.ratelimit-num-allowed-requests
+              name: argocd-image-updater-config
+              optional: true
```

Comment:
the above env var name should be consistent with the one used in run.go and webook.go.

the config map key should also be consistent with the rest as much as possible.